include/dpp/user.h

```diff
@@ -25,9 +25,9 @@
 #include <dpp/managed.h>
 #include <dpp/utility.h>
 #include <dpp/json_interface.h>
-
 namespace dpp {
-
+class cluster; //TODO Find a way to remove this forward declaration
```

Comment:
I would recommend completing this TODO yourself or removing it entirely, presumably the feature you're willing to add is complete and not half-baked.